ledger-tool/src/main.rs

```diff
@@ -818,8 +854,35 @@ fn open_blockstore(
         },
     ) {
         Ok(blockstore) => blockstore,
+        Err(BlockstoreError::RocksDb(err))
+            if (err
+                .to_string()
+                .starts_with("Invalid argument: Column family not found:")
+                || err
+                    .to_string()
+                    .starts_with("IO error: No such file or directory:"))
+                && access_type == AccessType::Secondary =>
+        {
+            info!("Blockstore is missing column(s) and unable to create with Secondary access");
+            if !force_update_to_open {
+                info!("Use --force-update-to-open to allow blockstore to update");
```

Comment:
Probably `error` is a better log level for these messages, as they are only shown in case of error, and they provide useful information about how to possibly address the error.